crates/block-producer/Cargo.toml

```diff
@@ -33,6 +33,7 @@ ckb-crypto = "0.100.0"
 ckb-fixed-hash = "0.100.0"
 ckb-types = "0.100.0"
 ckb-chain-spec = "0.100.0"
+tikv-jemallocator = { version = "0.4.0", features = ["unprefixed_malloc_on_supported_platforms", "profiling"] }
```

Comment:
`[target.'cfg(all(not(target_env = "msvc"), not(target_os="macos")))'.dependencies]`

https://github.com/nervosnetwork/ckb/blob/3c66fdbd8795841dce10c759c9d27e220340bb22/Cargo.toml#L99

Reply:
And do not enable profiling by default, it affect the performance.

We can use a feature to switch it.

https://github.com/nervosnetwork/ckb/blob/3c66fdbd8795841dce10c759c9d27e220340bb22/Cargo.toml#L107